spec/models/assignment_spec.rb

```diff
@@ -92,43 +92,158 @@
   end
 
   describe ".for_presentation" do
-    let(:split) { FactoryBot.create(:split) }
+    it "filters for one reason of underlying scopes (decision overrides)" do
+      split = FactoryBot.create(:split, decided_at: Time.zone.now)
+      assignment = FactoryBot.create(:assignment, split: split, updated_at: 1.day.ago)
 
-    it "doesn't return assignments for finished splits" do
-      finished_split = FactoryBot.create(:split, name: "finished_split", finished_at: Time.zone.now)
-      FactoryBot.create(:assignment, split: finished_split)
+      expect(described_class.for_presentation).not_to include(assignment)
+    end
+
+    it "combines excluding_decision_overrides and for_active_splits (argless) with no app_build" do
+      allow(described_class).to receive(:excluding_decision_overrides).and_call_original
+      allow(described_class).to receive(:for_active_splits).and_call_original
+      expect(described_class).not_to receive(:for_app_build)
+
+      described_class.for_presentation
 
-      expect(described_class.for_presentation.map(&:split_name)).not_to include("finished_split")
+      expect(described_class).to have_received(:excluding_decision_overrides)
+      expect(described_class).to have_received(:for_active_splits).with(no_args)
```

Comment:
Otherwise, we just assert on what scopes the combined scopes call, while at least ensuring that they don't raise exceptions. Seems like the scope-equivalent of spying on your OO collaborators, which I do a fair bit in model specs in order to assert that the plumbing works in integration while doing the in-depth testing on units.